test/no-empty-alt-text.test.js

```diff
@@ -19,19 +19,15 @@ describe("GH003: No Empty Alt Text", () => {
         '<img alt="" src="https://user-images.githubusercontent.com/abcdef.png">',
         "<img alt='' src='https://user-images.githubusercontent.com/abcdef.png'>",
         '<img src="cat.png" alt="" /> <img src="dog.png" alt="" />',
-        '<img src="dog.png" />',
-        '<img alt src="dog.png" />',
-        "<img alt><img alt>",
```

Comment:
3 cases were removed here, but the `failedRules` count decreased by 4 (8 to 4). Is that intentional?

Reply:
Yes, this is expected. A little obscure but line 24 previously had two image tags: `"<img alt><img alt>"`